gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/driver/remote/AbstractRemoteGraphProvider.java

```diff
@@ -221,16 +221,22 @@ public abstract class AbstractRemoteGraphProvider extends AbstractGraphProvider
     private final Cluster cluster;
     private final Client client;
     private final boolean useComputer;
+    private final String queryLanguage;
 
 
     public AbstractRemoteGraphProvider(final Cluster cluster) {
         this(cluster, false);
     }
 
     public AbstractRemoteGraphProvider(final Cluster cluster, final boolean useComputer) {
+        this(cluster, useComputer, "gremlin-groovy");
```

Comment:
i guess the `AbstractRemoteGraphProvider` tests using "gremlin-groovy" will test the groovy path for remote execution, i.e. taking `GremlinLang` produced Gremlin, passing it through the translator to groovy so that it can be executed in the `GremlinGroovyScriptEngine`. i think it's worth clarifying that in the class javadocs. worth tracking a better mechanism for testing `GremlinGroovyScriptEngine` that doesn't rely on the java tests and instead uses the feature tests.

Reply:
I added javadoc to constructor